package.json

```
@@ -153,33 +185,149 @@
+          "default": [],
+          "markdownDescription": "Configure connection(s) to [SonarQube](https://sonarqube.org). Don't forget to also configure the project binding in `#sonarlint.connectedMode.project#`.\n\nExample:\n\n    \"sonarlint.connectedMode.connections.sonarqube\": [\n      {\n        \"serverUrl\": \"https://sonar.mycompany.com\",\n        \"token\": \"V2VkIE1...\"\n      }\n    ]\n\nSpecify a `connectionId` if you want to define multiple connections.",
+          "examples": [
+            {
+              "serverUrl": "https://<myServerUrl>",
+              "token": "<mySQToken>"
+            },
+            {
+              "connectionId": "<myConnectionId>",
+              "serverUrl": "https://<myServerUrl>",
+              "token": "<mySQToken>"
+            }
+          ],
+          "items": {
+            "type": "object",
+            "properties": {
+              "connectionId": {
+                "type": "string",
+                "description": "A unique identifier for this connection to be used as a reference in `#sonarlint.connectedMode.project#`. Only needed if you plan to use multiple connections to SonarQube/SonarCloud.",
+                "default": "default"
+              },
+              "serverUrl": {
+                "type": "string",
+                "description": "URL of the server."
+              },
+              "token": {
+                "type": "string",
+                "description": "Token generated from My Account>Security in SonarQube"
+              }
+            },
+            "additionalProperties": false,
+            "required": [
+              "serverUrl",
+              "token"
+            ]
+          }
+        },
+        "sonarlint.connectedMode.connections.sonarcloud": {
+          "type": "array",
+          "scope": "application",
+          "default": [],
+          "markdownDescription": "Configure connection(s) to [SonarCloud](https://sonarcloud.io). Don't forget to also configure the project binding in `#sonarlint.connectedMode.project#`.\n\nExample:\n\n    \"sonarlint.connectedMode.connections.sonarcloud\": [\n      {\n        \"organizationKey\": \"myOrg\",\n        \"token\": \"V2VkIE1...\"\n      }\n    ]\n\nSpecify a `connectionId` if you want to define multiple connections.",
+          "examples": [
+            {
+              "organizationKey": "<myOrgKey>",
+              "token": "<mySCToken>"
+            },
+            {
+              "connectionId": "<myConnectionId>",
+              "organizationKey": "<myOrgKey>",
+              "token": "<mySCToken>"
             }
+          ],
+          "items": {
+            "type": "object",
+            "properties": {
+              "connectionId": {
+                "type": "string",
+                "description": "A unique identifier for this connection to be used as a reference in `#sonarlint.connectedMode.project#`. Only needed if you plan to use multiple connections to SonarQube/SonarCloud.",
```

```suggestion
                "description": "A unique identifier for this connection to be used as a reference in `#sonarlint.connectedMode.project#`. Only needed if you plan to use multiple connections to SonarCloud.",
```
It might be worth mentioning that several connections are needed to bind to different organizations.

---

See my previous comment about SQ/SC. I'll add something about mutliple orgs.